### PR TITLE
Updates to Roadmap

### PR DESCRIPTION
**Pull request by goslingo**
- Set restricted mainnet as the default position when the roadmap first loads
- Hide and show next and previous arrows at start and end of scroll accordingly (in mobile view)

### CODE REVIEW

**Comment by goslingo** [comment]
@pennyandrews changes as discussed, ready to go

**Comment by Ben-peoplemake** [comment]
Performing some testing on the changes here are some issues that I have noticed:


1. Sometimes when **clicking** on a planet either side of the one current focussed I am "scrolled" in the opposite direction to what I clicked. (Hopefully the video recording uploaded makes it clear. _(The click and drag works)_
https://user-images.githubusercontent.com/117179940/201908582-8f3920b7-5573-4c68-b565-4b1b36cdfaac.mov

2. This may be dumb, so apologies. We don't appear to have any translations for the roadmap section, but we do in some of the other sections of the site. I'm new so not aware how we get translations etc or if it's something we need before rolling out the changes.

**Comment by pennyandrews** [comment]
@Ben-peoplemake 

> 2. This may be dumb, so apologies. We don't appear to have any translations for the roadmap section, but we do in some of the other sections of the site. I'm new so not aware how we get translations etc or if it's something we need before rolling out the changes.

Translations are contributed in a decentralised way via community contribution, each language supported by a named Vega Ambassadors (there is a little about it in the [readme](https://github.com/vegaprotocol/vega.xyz/#readme)). So we have no translations currently for Russian or Chinese and some but not all for Spanish but they are trickling in - you'll see there are a few open PRs to add translations currently. Hit me up on Slack if you need more info on this.

**Comment by pennyandrews** [comment]
@goslingo have you managed to have a look at Ben's comment 1. above? would be great to resolve this if it's not too time consuming.

**Comment by pennyandrews** [comment]
What's the latest on this PR @goslingo ? Can we get it live?

**Comment by goslingo** [comment]
@pennyandrews ready for review:

https://deploy-preview-319--vega.netlify.app/
https://deploy-preview-319--vega.netlify.app/use
https://deploy-preview-319--vega.netlify.app/key-concepts

File: src/content/roadmap/03/content.cn.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-step_title: H2 2022
+step_title: <strike>2022 H2</strike> <i>any week now!</i><br><a class="underline hover:no-underline" target="_blank" href="https://github.com/vegaprotocol/vega/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22%F0%9F%A4%A0+Oregon+Trail%22">See what's left</a>
```
Comment (pennyandrews) [inline src/content/roadmap/03/content.cn.md:2]:
```suggestion
step_title: <strike>2022 H2</strike> <i>any week now!</i><br><a class="underline hover:no-underline" target="_blank" href="https://github.com/orgs/vegaprotocol/projects/125/views/1">See what's left</a>
```

**Review by pennyandrews — CHANGES_REQUESTED** [review]
Just the one suggested change, otherwise this is good to go!

**Review by pennyandrews — APPROVED** [review]
Change approved! ;)

**Pull request merged**